scripts/serverSetup.md

```diff
@@ -0,0 +1,66 @@
+# Getting started with YesWorkflow Web Components
+
+**Please ensure that JDK 10 and Maven 3+ are installed on your machine before continuing and confirm that JDK 10 is your default JAVA_HOME**
+
+## Deploying the web sever
+In the yw_website directory, change directory into the scripts directory. You should see a number of bash scripts:
+```
+deploy_python.sh
+deploy_python3.sh
+remote_deploy_python.sh
+remote_deploy_python3.sh
+workflow_sample.gv
+```
+If you would like to deploy a test server on your localhost, you may run one of these two files:
+```
+deploy_python.sh
+deploy_python3.sh
+```
+For windows users, use deploy_python.sh
+
+For Mac/Linux users, use deploy_python3.sh
+
+These scripts will deploy a web server running on your local machine at 
+```http://localhost:8000```
+
+Deploying on a remote web server is a bit trickier because we will need
+to edit our remote deploy scripts. Let's take a look at ```remote_deploy_python3.sh```
```

Comment:
just remove the remote deploy stuff, the script is too personalized to my stuff, I need to just remove them